pkg/logger/logger.go

```diff
@@ -98,6 +101,30 @@ func NewReceptorLogger(prefix string) *ReceptorLogger {
 	}
 }
 
+func NewReceptorLoggerWithSuffix(prefix string, suffix map[string]interface{}) *ReceptorLogger {
+	logger := NewReceptorLogger(prefix)
+	logger.SetSuffix(suffix)
+
+	return logger
+}
+
+func (rl *ReceptorLogger) SetSuffix(suffix map[string]interface{}) {
```

Comment:
Can we add a comment to this function so that developers know when to use `SetSuffix` and `UpdateSuffix`?